tests/sentry/integrations/slack/test_link_team.py

```diff
@@ -87,8 +103,18 @@ def get_linked_teams(
 
     def _create_user_with_valid_role_through_team(self):
         user = self.create_user(email="foo@example.com")
-        admin_team = self.create_team(org_role="admin")
-        self.create_member(organization=self.organization, user=user, teams=[admin_team])
+        self.team.update(org_role="admin")
```

Comment:
In the view, we respond with the id of `self.team`. Because we are already creating a team in `setUp`, this will error when we create another team b/c the id of the team does not match. We get around this by directly updating `self.team` instead of creating a new team